### ISSUE TITLE
path is malformatted when starting preview

### ISSUE THREAD

**Issue by joshuajeschek** [dendronhq/dendron#2325]
## Please select the area your bug applies to. (Multiple selections are Possible. You can leave blank if you're not sure.)

- [x] Views.  Dendron Preview, Tree View, Side Panels in the UI

## Describe the bug

When I start a a preview of a note, an error occurs and the following is to be found in the Dendron output:
```json
{
    "ctx": "dendron.showPreview:run",
    "error": {
        "stack": "Error: ENOENT: no such file or directory, open 'C:\\c:\\Users\\joshu\\Documents\\dendron\\tuc\\mathematik-iii.algebraische-strukturen.md'",
        "message": "error while running command: ENOENT: no such file or directory, open 'C:\\c:\\Users\\joshu\\Documents\\dendron\\tuc\\mathematik-iii.algebraische-strukturen.md'",
        "name": "DendronError",
        "status": "unknown",
        "payload": "{}",
        "innerError": {
            "errno": -4058,
            "code": "ENOENT",
            "syscall": "open",
            "path": "C:\\c:\\Users\\joshu\\Documents\\dendron\\tuc\\mathematik-iii.algebraische-strukturen.md",
            "vslsStack": []
        }
    }
}
```

! this started happening right after upgrading to 0.80

## To Reproduce

1. open any file in the editor
2. start the preview

## Expected behavior

The preview should be shown

## Screenshots

![image](https://user-images.githubusercontent.com/64850647/152050386-6bd91c9e-7557-4dce-9958-84c8dc11f058.png)

## Desktop (please complete the following information)

- OS: Win11
- Browser: []
- Version: 0.80

## Additional context

! this started happening right after upgrading to 0.80

## Dendron Log file

Please attach the output of `> Dev:Dendron: Open Logs` here
```json
{
    "level": 30,
    "time": 1643748912094,
    "pid": 24516,
    "hostname": "JOSHS-SPECTRE",
    "name": "dendron",
    "ctx": "dendron.showPreview:run",
    "msg": "pre-execute"
}
{
    "level": 50,
    "time": 1643748912097,
    "pid": 24516,
    "hostname": "JOSHS-SPECTRE",
    "name": "dendron",
    "ctx": "dendron.showPreview:run",
    "error": {
        "stack": "Error: ENOENT: no such file or directory, open 'C:\\c:\\Users\\joshu\\Documents\\dendron\\tuc\\mathematik-iii.algebraische-strukturen.md'",
        "message": "error while running command: ENOENT: no such file or directory, open 'C:\\c:\\Users\\joshu\\Documents\\dendron\\tuc\\mathematik-iii.algebraische-strukturen.md'",
        "name": "DendronError",
        "status": "unknown",
        "payload": "{}",
        "innerError": {
            "errno": -4058,
            "code": "ENOENT",
            "syscall": "open",
            "path": "C:\\c:\\Users\\joshu\\Documents\\dendron\\tuc\\mathematik-iii.algebraische-strukturen.md",
            "vslsStack": []
        }
    }
}
{
    "level": 30,
    "time": 1643748926320,
    "pid": 24516,
    "hostname": "JOSHS-SPECTRE",
    "name": "dendron",
    "ctx": "WindowWatcher:onDidChangeActiveTextEditor",
    "editor": "c:\\Users\\joshu\\Documents\\dendron\\tuc\\mathematik-iii.algebraische-strukturen.md"
}
```

**Comment by Harshita-mindfire** [comment]
Hello @joshuajeschek, Thanks for the report! This should be fixed now in Dendron version 0.80.2
Feel free to re-open if not the case

**Comment by joshuajeschek** [comment]
Thanks looking forward to it :)
